www/src/components/chat/chat-list.tsx

```diff
@@ -130,7 +136,7 @@ export const ChatList: React.FC = React.memo(() => {
         sendMessage(editedMessage);
       }
     },
-    [selectedChatId, sendMessage, currentMessages],
+    [selectedChatId, sendMessage, currentMessages]
```

Comment:
lint issue, perhaps we should just change to using prettier/eslint and forget biome since everyone's vscode uses prettier? 😅 wdyt?

Reply:
I changed my json settings to use biome but we could just have a .vscode for this project

Reply:
Ohh sorry, I didn't configure to use the biome formatter, for me the most relevant is to standardized everything, so if everyone is already using it, I'll configure to use Biome